modules/installation-custom-ibm-cloud-vpc.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-ibm-cloud-vpc.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-ibm-cloud-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into the subnets of an existing IBM Virtual Private Cloud (VPC). By deploying {product-title} into an existing VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
+
+Because the installation program cannot know what other components are also in your existing subnets, it cannot choose subnet CIDRs and so forth on your behalf. You must configure networking for the subnets that you install your cluster to yourself.
+
+[id="installation-custom-ibm-cloud-vpc-requirements_{context}"]
+== Requirements for using your VPC
+
+The union of the VPC CIDR block and the machine network CIDR must be non-empty. The subnets must be within the machine network.
```

Comment:
While I agree with spelling first instances of acronyms, CIDR is said so frequently that it has become its own word. It would be (almost) awkward to actually spell it out as Classless Inter-Domain Routing.